eng/common/pipelines/templates/steps/publish-blobs.yml

```diff
@@ -10,8 +10,10 @@ parameters:
 steps:
 - template: /eng/common/pipelines/templates/steps/set-default-branch.yml
 - pwsh: |
-    Invoke-WebRequest -MaximumRetryCount 10 -Uri "https://aka.ms/downloadazcopy-v10-windows" -OutFile "azcopy.zip" | Wait-Process;
-    Expand-Archive -Path "azcopy.zip" -DestinationPath "$(Build.BinariesDirectory)/azcopy/"
+    if (!(Test-Path '$(Build.BinariesDirectory)/azcopy/azcopy_windows_amd64_*/azcopy.exe')) {
```

Comment:
While this will likely help the immediate issue, we can probably improve this a little further by checking to see if azcopy is already on the path via gcm because I believe it is already on the agents now.